python/pyspark/sql/column.py

```diff
@@ -303,8 +333,25 @@ def isin(self, *cols):
     desc = _unary_op("desc", "Returns a sort expression based on the"
                              " descending order of the given column name.")
 
-    isNull = _unary_op("isNull", "True if the current expression is null.")
-    isNotNull = _unary_op("isNotNull", "True if the current expression is not null.")
+    _isNull_doc = ''' True if the current expression is null. Often combined with 
+                      :func:`DataFrame.filter` to select rows with null values.
+
+                      >>> df2.collect()
+                      [Row(name=u'Tom', height=80), Row(name=u'Alice', height=None)]
+                      >>> df2.filter( df2.height.isNull ).collect()
+                      [Row(name=u'Alice', height=None)]
+                  '''
+    _isNotNull_doc = ''' True if the current expression is null. Often combined with 
```

Comment:
Do you mean formatting it more like https://github.com/map222/spark/blob/90d1150867184d86b029c1f6397dcd855b1f5961/python/pyspark/sql/column.py#L482-L493?